### PR TITLE
Add Elasticsearch-2022-03-02  webinar contents

### PR DESCRIPTION
**Pull request by raihankhan**
Signed-off-by: raihankhan <raihan@appscode.com>

### CODE REVIEW

File: content/post/elasticsearch-webinar-2022.03.02/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Later in this webinar, They discussed `Kibana` support for `Dashboard`. Where they created an `ElasticsearchDashboard` instance to deploy `Kibana`. They showed lots of customizations for deploying `ElasticsearchDashboard`. They also created an `Index Lifecycle Management` policy through kibana for Elasticsearch Hot-Warm-Cold Architecture. An `Index Template` was created to configure elasticsearch indices. They deployed `Logstash` and `Filebeat` for propagating logs to elasticsearch. After that, they showed a demo of monitoring kubernetes container logs from kibana. Finally, they performed some manual testing to ensure the Elasticsearch Hot-Warm-Cold architecture is implemented, and it is functioning as the `ILM` policy is set to be.
+
+
+  Take a deep dive into the full webinar below:
+
+<iframe width="800" height="500" src="https://youtu.be/R-eYc2cUXQY" title="Elasticsearch Hot-Warm-Cold Architecture Management with Kibana in Kubernetes" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (kamolhasan) [inline content/post/elasticsearch-webinar-2022.03.02/index.md:43]: https://youtu.be/R-eYc2cUXQY?t=1

File: content/post/elasticsearch-webinar-2022.03.02/index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Later in this webinar, They discussed `Kibana` support for `Dashboard`. Where they created an `ElasticsearchDashboard` instance to deploy `Kibana`. They showed lots of customizations for deploying `ElasticsearchDashboard`. They also created an `Index Lifecycle Management` policy through kibana for Elasticsearch Hot-Warm-Cold Architecture. An `Index Template` was created to configure elasticsearch indices. They deployed `Logstash` and `Filebeat` for propagating logs to elasticsearch. After that, they showed a demo of monitoring kubernetes container logs from kibana. Finally, they performed some manual testing to ensure the Elasticsearch Hot-Warm-Cold architecture is implemented, and it is functioning as the `ILM` policy is set to be.
+
+
+  Take a deep dive into the full webinar below:
+
+<iframe width="800" height="500" src="https://youtu.be/R-eYc2cUXQY?t=1" title="Elasticsearch Hot-Warm-Cold Architecture Management with Kibana in Kubernetes" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (kamolhasan) [inline content/post/elasticsearch-webinar-2022.03.02/index.md:43]: src="https://www.youtube.com/embed/R-eYc2cUXQY"

**Review by kamolhasan — CHANGES_REQUESTED** [review]

File: content/post/elasticsearch-webinar-2022.03.02/index.md (hunk 2 above)
Comment (raihankhan) [reply content/post/elasticsearch-webinar-2022.03.02/index.md]: updated

**Review by kamolhasan — APPROVED** [review]
LGTM!

**Pull request merged**